sdk/python/pulumi_aws/kinesis/_inputs.py

```diff
@@ -0,0 +1,3991 @@
+    def deserializer(self) -> pulumi.Input['FirehoseDeliveryStreamExtendedS3ConfigurationDataFormatConversionConfigurationInputFormatConfigurationDeserializerArgs']:
+        """
+        Nested argument that specifies which deserializer to use. You can choose either the Apache Hive JSON SerDe or the OpenX JSON SerDe. More details below.
+        """
+        return pulumi.get(self, "deserializer")
+
+    @deserializer.setter
+    def deserializer(self, value: pulumi.Input['FirehoseDeliveryStreamExtendedS3ConfigurationDataFormatConversionConfigurationInputFormatConfigurationDeserializerArgs']):
+        pulumi.set(self, "deserializer", value)
+
+
+@pulumi.input_type
+class FirehoseDeliveryStreamExtendedS3ConfigurationDataFormatConversionConfigurationInputFormatConfigurationDeserializerArgs:
+    def __init__(__self__, *,
+                 hive_json_ser_de: Optional[pulumi.Input['FirehoseDeliveryStreamExtendedS3ConfigurationDataFormatConversionConfigurationInputFormatConfigurationDeserializerHiveJsonSerDeArgs']] = None,
+                 open_xjson_ser_de: Optional[pulumi.Input['FirehoseDeliveryStreamExtendedS3ConfigurationDataFormatConversionConfigurationInputFormatConfigurationDeserializerOpenXJsonSerDeArgs']] = None):
+        """
+        :param pulumi.Input['FirehoseDeliveryStreamExtendedS3ConfigurationDataFormatConversionConfigurationInputFormatConfigurationDeserializerHiveJsonSerDeArgs'] hive_json_ser_de: Nested argument that specifies the native Hive / HCatalog JsonSerDe. More details below.
+        :param pulumi.Input['FirehoseDeliveryStreamExtendedS3ConfigurationDataFormatConversionConfigurationInputFormatConfigurationDeserializerOpenXJsonSerDeArgs'] open_xjson_ser_de: Nested argument that specifies the OpenX SerDe. More details below.
+        """
+        if hive_json_ser_de is not None:
+            pulumi.set(__self__, "hive_json_ser_de", hive_json_ser_de)
+        if open_xjson_ser_de is not None:
+            pulumi.set(__self__, "open_xjson_ser_de", open_xjson_ser_de)
+
+    @property
+    @pulumi.getter(name="hiveJsonSerDe")
+    def hive_json_ser_de(self) -> Optional[pulumi.Input['FirehoseDeliveryStreamExtendedS3ConfigurationDataFormatConversionConfigurationInputFormatConfigurationDeserializerHiveJsonSerDeArgs']]:
+        """
+        Nested argument that specifies the native Hive / HCatalog JsonSerDe. More details below.
+        """
+        return pulumi.get(self, "hive_json_ser_de")
+
+    @hive_json_ser_de.setter
+    def hive_json_ser_de(self, value: Optional[pulumi.Input['FirehoseDeliveryStreamExtendedS3ConfigurationDataFormatConversionConfigurationInputFormatConfigurationDeserializerHiveJsonSerDeArgs']]):
+        pulumi.set(self, "hive_json_ser_de", value)
+
+    @property
+    @pulumi.getter(name="openXJsonSerDe")
+    def open_xjson_ser_de(self) -> Optional[pulumi.Input['FirehoseDeliveryStreamExtendedS3ConfigurationDataFormatConversionConfigurationInputFormatConfigurationDeserializerOpenXJsonSerDeArgs']]:
```

Comment:
Ideally this would be `open_x_json_ser_de`. Fix: https://github.com/pulumi/pulumi/pull/5202

Reply:
We'll need to update the `go.mod` require for pulumi/pulumi in this PR to reference this commit: https://github.com/pulumi/pulumi/commit/62925433421370e32e614f2d2647f577673148bc